routes/views/donate.js

```diff
@@ -1,39 +1,183 @@
 var keystone = require('keystone');
 var Donation = keystone.list('Donation');
+var SupporterIndividual= keystone.list('SupporterIndividual');
+
+// paypal configuration set to test interface created by Andreas
+var paypal = require('paypal-rest-sdk');
+var config = {
+  "port" : 5000,
+  "api" : {
+		"mode": "sandbox", //sandbox or live
+    // "host" : "api.sandbox.paypal.com",
+    // "port" : "",
+    "client_id" : "ATa2OklO_gXCaV_RwJ_NpKlSOkegl2gG9JWAIJdj17NUBe9H5v8JZ7ZQRb00SYdT17VBbdXy9J_pV5Bf",  // dev (meyer) paypal application client id
+    "client_secret" : "EOk-VLq_nhGCsKILSiDLkGEVOvEz4Y4yDQiX1T7eYQXhg0kGOq85Ywmi2SjA6ak4M89x2TLQTx2vWUQS" // dev (meyer) paypal application secret id
```

Comment:
Be sure to remove these from source code (i know they're sandbox but still)
use `.env` file in root directory to list environment variables and they will be available here
```
#.env
PAYPAL_CLIENT_ID=foo
PAYPAL_CLIENT_SECRET=bar

#donation.js
...
"client_id": process.env.PAYPAL_CLIENT_ID,
...
```